### ISSUE TITLE
MonoDevelop on Linux can not see external hdd

### ISSUE THREAD

**Issue by eredacokmerke** [mono/monodevelop#3763]
I installed MonoDevelop 7.3.3 (build 5) on my Manjaro Linux 4.15. I clicked New -> .NET -> Console Project then Next. I clicked Browse button for location. I want to select my external hdd but MonoDevelop can not see inside of hdd. the location of hdd is /run/media/user/6BD63F8528DF4F7A. But there is no media folder in /run in MonoDevelop Browse screen. So I can not select hdd.

**Comment by jstedfast** [comment]
Hmmm, that should be handled by the GtkFileChooser dialog code which is part of Gtk and not MonoDevelop.

Do other Gtk applications see it?

Are you sure the hdd is mounted (in the right location)?

**Comment by eredacokmerke** [comment]
yes, I am sure hdd is mounted and the path is right. I can open hdd by using nautilus (gnome file manager).  I tried [bless](https://github.com/bwrsandman/Bless) and I can open external hdd.

**Comment by directhex** [comment]
Interesting issue. GIO/GVFS provides sufficient magic to allow access to USB drives to the sandbox, and I have that working locally fine. That's with the drives mounted in /media. Support for /run/media was supposedly added to GLib around version 2.32, and we're using much newer, so it'll take me some time to instantiate an environment for testing where /run is used.
